### PR TITLE
Dark mode: NUX improvements

### PR DESCRIPTION
**Pull request by frosty**
Refs #12320. This PR implements some improvements to the appearance of the login flow for dark mode.

* Updated primary and secondary button styles
* Fixed invisible text in text fields, and increased padding between icon and text
* Selectively show dark or light Sign in with Apple button based on user interface style

dark-before-after

Shouldn't be noticeable changes in light mode.

**To test:**

* Build and run in iOS 13, light and dark mode, and in iOS 12
* Check things look as expected

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

cc @mattmiklic

### CODE REVIEW

File: WordPress/Classes/Extensions/Colors and Styles/UIColor+MurielColors.swift [hunk 1]
```diff
@@ -258,6 +258,37 @@ extension UIColor {
...
     static var primaryButtonDownBackground = muriel(color: .accent, .shade80)
     static var primaryButtonDownBorder = muriel(color: .accent, .shade90)
+
+    static var secondaryButtonBackground: UIColor {
+        #if XCODE11
+        if #available(iOS 13, *) {
+            return UIColor(light: .white, dark: .systemGray5)
+        }
+        #endif
+        return UIColor(light: .white, dark: .neutral(.shade80))
```
Comment (nheagy) [inline WordPress/Classes/Extensions/Colors and Styles/UIColor+MurielColors.swift]: We don't need dark mode colors if this isn't in Xcode 11. You can return `.white` here. Same principle applies to the next three.

**Comment by frosty** [comment]
Thanks @nheagy!

**Comment by frosty** [comment]
@nheagy I fixed the issue you mentioned, and I noticed the NUX epilogue also needed some love so I pushed up some fixes there – would you mind checking that too please?

login-nux

**Comment by mattmiklic** [comment]
> Updated primary and secondary button styles
> Fixed invisible text in text fields, and increased padding between icon and text
> Selectively show dark or light Sign in with Apple button based on user interface style

All of this looks good! There are some additional updates to be made to the prologue and the button styles but I think we can address them in separate PRs.

**Pull request merged**